youtokentome/yttm_cli.py

```diff
@@ -98,7 +98,10 @@ def bpe(data, model, vocab_size, coverage, n_threads, pad_id, unk_id, bos_id, eo
 @click.option(
     "--stream", is_flag=True, help="Process each line before reading the next one."
 )
-def encode(model, output_type, n_threads, bos, eos, reverse, stream):
+@click.option(
+    "--dropout_prob", type=click.FLOAT, default=0, show_default=True, help="Process each line before reading the next one."
```

Comment:
it may be helpful to add the link on the paper (https://arxiv.org/abs/1910.13267) in help.

Reply:
And it seems like current help contains unrelated information

Reply:
Also, I would recommend adding callback on dropout option to throw an exception if the value is not within [0, 1] range
https://click.palletsprojects.com/en/7.x/options/#callbacks-for-validation

Reply:
> Also, I would recommend adding callback on dropout option to throw an exception if the value is not within [0, 1] range

We already do it here https://github.com/VKCOM/YouTokenToMe/pull/45/files#diff-fe449cd06a471c4e41e1deae980a3badR92

Reply:
ok then